Library/Formula/v8.rb

```diff
@@ -14,6 +14,7 @@ class V8 < Formula
   end
 
   option "with-readline", "Use readline instead of libedit"
+  option "with-icu4c", "Use icu4c for l18n support"
```

Comment:
Can probably drop this line. It's fairly obvious what you'd use an `icu4c` dep for.